ngrinder-controller/src/main/java/org/ngrinder/perftest/repository/PerfTestRepository.java

```diff
@@ -60,7 +60,7 @@
 	 * @param region region where the test belong to
 	 * @return perf test list
 	 */
-	List<PerfTest> findAllByStatusAndRegionOrderByScheduledTimeAsc(Status status, String region);
+	List<PerfTest> findAllByStatusAndRegionStartsWithOrderByScheduledTimeAsc(Status status, String region);

```

Comment:
StartsWithOrder??

Reply:
After add subregion. region is stored to the database like `{region}.{subregion}`.
So I use StartsWith to select all of the perftests that have configured region or subregion from the database.